src/components/TextEditor/components/ShareSketchModal.js

```diff
@@ -13,50 +13,50 @@ import '../../../styles/Modals.scss';
  * ShareSketchModal is a full-screen modal that displays
  * the current program's view-only route, which the user can manually
  * copy or use the "Copy To Clipboard" Button
- * @param {Boolean} showModal modal render state
- * @param {Function} toggleModal function that toggles the modal's render state
- * @param {String} shareUrl the URL to display/copy to clipboard
+ * showModal modal render state
+ *  toggleModal function that toggles the modal's render state
+ *  shareUrl the URL to display/copy to clipboard
  */
-class ShareSketchModal extends React.Component {
-  state = {
-    copyStatus: 'Hit "Copy to Clipboard"!',
-  };
+const ShareSketchModal = function (props) {
+  const { shareUrl, showModal, toggleModal } = props;
+  const [str, setStr] = useState([{ copyStatus: 'Hit "Copy to Clipboard"!' }]);
```

Comment:
Can we instead make this useState called 'copyStatus' and 'setCopyStatus' that stores a string.  You may have to also change the lines that call this later on.